docs/design/2022-12-21-auto-reload-tls-certificate.md

```diff
@@ -0,0 +1,75 @@
+# Auto reload tls certificate for TiFlash
+
+- Author: [Weiqi Yan](https://github.com/ywqzzy)
+
+## Introduction
+
+In TiFlash config, we can set certificate as follows:
+```YAML
+[security]
+ca_path = "/path/to/tls/ca.crt"
+cert_path = "/path/to/tls/tiflash.crt"
+key_path = "/path/to/tls/tiflash.pem"
+```
+Then the TiFlash Server can use Tls certificates to enable secure transmission.
+
+Since the tls certificate has a valid period, in order not to affect the normal operation of online business, the TiFlash node should not be restarted manually when replacing the certificate, so it needs to support automatic rotation of tls certificates.
```

Comment:
```suggestion
Since the TLS certificate has a valid period, in order not to affect the normal operation of online business, the TiFlash node should not be restarted manually when replacing the certificate, so it needs to support automatic rotation of TLS certificates.
```